<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tags</class>
 <widget class="QWidget" name="Tags">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>&amp;Tags</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="2,0,0,0,0,0,2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="buttonAssignedTagsAdd"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="buttonAssignedTagsDelete">
           <property name="text">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="buttonAssignedTagsClear">
           <property name="text">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelAssignedTags">
         <property name="text">
          <string>Assigned tags</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="listAssignedTags">
         <property name="contextMenuPolicy">
          <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDropMode::DragDrop</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::DropAction::MoveAction</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
         </property>
         <property name="isWrapping" stdset="0">
          <bool>true</bool>
         </property>
         <property name="uniformItemSizes">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAssignedTagsAdd">
   <property name="icon">
    <iconset resource="../../cmake-build-relwithdebinfo/src/icons.qrc">
     <normaloff>:/icons/bx-plus-circle.svg</normaloff>:/icons/bx-plus-circle.svg</iconset>
   </property>
   <property name="text">
    <string>Assign new tag</string>
   </property>
   <property name="toolTip">
    <string>Assign a new tag</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAssignedTagsDelete">
   <property name="icon">
    <iconset resource="../../cmake-build-relwithdebinfo/src/icons.qrc">
     <normaloff>:/icons/bx-minus-circle-red.svg</normaloff>:/icons/bx-minus-circle-red.svg</iconset>
   </property>
   <property name="text">
    <string>Delete assigned tag</string>
   </property>
   <property name="toolTip">
    <string>Delete a tag from the assigned list</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAssignedTagsClear">
   <property name="icon">
    <iconset resource="../../cmake-build-relwithdebinfo/src/icons.qrc">
     <normaloff>:/icons/bx-eraser-red.svg</normaloff>:/icons/bx-eraser-red.svg</iconset>
   </property>
   <property name="text">
    <string>Clear all assigned tags</string>
   </property>
   <property name="toolTip">
    <string>Clear all assigned tags</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../cmake-build-relwithdebinfo/src/icons.qrc"/>
 </resources>
 <connections/>
</ui>
